website/views/shared/mixins.jade

```diff
@@ -11,3 +11,9 @@ mixin aLink(url, label)
     a(href="", ng-click="externalLink('#{url}')")= label
   else
     a(href='#{url}', target='_blank')= label
+
+mixin previewMarkdown(text)
+  .panel.panel-warning
+    .panel-heading=env.t('msgPreviewHeading')
+    .panel-body.markdown-preview
+      markdown(text='#{text}')
```

Comment:
If we're not mocking the look of the task, we should at least show the notes as markdown.